common/lib/xmodule/xmodule/templates/problem/imageresponse.yaml

```diff
@@ -4,19 +4,33 @@ metadata:
     markdown: !!null
```

Comment:
@lamagnifica @srpearce, as this problem is currently engineered, it is not accessible :(  However, there is no reason why such a problem type can't be made accessible.  This problem type needs to be re-engineered.